openadapt/utils.py

```diff
@@ -15,21 +15,20 @@
 import mss.base
 import numpy as np
 
-from openadapt import common
+from openadapt import common, config, scrub
 
 
-# TODO: move to config.py
-DIRNAME_PERFORMANCE_PLOTS = "performance"
-
 EMPTY = (None, [], {}, "")
 
 
-def configure_logging(logger, log_level):
+def configure_logging(logger, log_level, scrub_logging):
     log_level_override = os.getenv("LOG_LEVEL")
```

Comment:
Please add a TODO like so:

```
# TODO: redact log messages (https://github.com/Delgan/loguru/issues/17#issuecomment-717526130)
```

Also please remove scrub_logging here and everywhere else

Reply:
Done!